vhost-user-backend/Cargo.toml

```diff
@@ -10,10 +10,12 @@ license = "Apache-2.0"
 
 [features]
 xen = ["vm-memory/xen", "vhost/xen"]
+postcopy = ["vhost/postcopy", "userfaultfd"]
```

Comment:
IMHO we can enable this by default.
Do we really need the feature in `vhost` too? At the end we added just some wrappers right?

Reply:
For `vhost-user-backend` I think having it as opt-in rather then default makes sense. This is quite specific protocol part + requires additional dependency.
For `vhost`, I guess we can remove feature gate and nothing will break because changes are additive. But then we have dis balance between `vhost` and `vhost-user-backend` crates.

Reply:
> For `vhost-user-backend` I think having it as opt-in rather then default makes sense. This is quite specific protocol part + requires additional dependency.

Ack, makes sense.

> For `vhost`, I guess we can remove feature gate and nothing will break because changes are additive. But then we have dis balance between `vhost` and `vhost-user-backend` crates.

I don't have a strong opinion, so I'm fine also to have it in `vhost`.

I just noticed that `coverage` now increased. I think because previously the new code was enabled by default and now it is not. Eventually we need to enable the feature in `crate_features` in `coverage_config_x86_64.json`, but if we don't know how to test it, maybe just adjust the coverage value for now.

Reply:
I will adjust coverage after all rebasing is done.

Reply:
@ShadowCurse FYI #229 is now merged